app/src/main/java/xterminators/spellingbee/GuiController.java

```diff
@@ -0,0 +1,126 @@
+package xterminators.spellingbee;
+
+import java.io.*;
+import java.awt.*;
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.nio.file.Paths;
+
+public class GuiController {
+
+    private JFrame m_frame;
+    private JPanel m_panel;
+    private JTextField tbGuess;
```

Comment:
It's kind of difficult to tell what this variable is at first glance, is it possible to give it a more descriptive (even if slightly longer) name?